### PR TITLE
Fixes cloners and portachems not working right with tier 4

### PR DESCRIPTION
**Pull request by Iamgoofball**
coulda sworn I fixed these

### CODE REVIEW

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -44,6 +44,8 @@
 	for(var/obj/item/weapon/stock_parts/manipulator/P in component_parts)
 		speed_coeff += P.rating
 	heal_level = (efficiency * 15) + 10
+	if(heal_level > 100)
```
Comment (RemieRichards) [inline code/game/machinery/cloning.dm]: fuck off with this shit, you know how to properly cap a number.

**Comment by Razharas** [comment]
Also theres never a reason to upgrade manipulators to level 4, not nice

**Comment by Kromgar** [comment]
I'd just like this to be merged so we can clone people at t4 already

**Comment by Iamgoofball** [comment]
@Razharas how so?

**Comment by Razharas** [comment]
Disregard this i didnt check the refresh parts proc

**Pull request merged**